tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -71,14 +72,15 @@ describe("List Collector Repeating Blocks", () => {
       await click(AnyOtherTradingDetailsPage.submit());
       await click(SectionCompaniesPage.submit());
       await click(SubmitPage.submit());
+      await expect(await browser.getUrl()).toContain(ThankYouPage.pageName);
     });
   });
 
-  describe("Given a journey through the list collector with repeating blocks, the companies can be added, removed and edited.", () => {
+  describe("Given a journey through the list collector with repeating blocks", () => {
```

Comment:
Same comment - would we want this to be something like `A journey with list edits through a list collector with repeating blocks` to give a bit of context but not overdo it?

Reply:
Amended slightly now 👍